### PR TITLE
[release-ocm-2.8] MGMT-18332: Use centos stream 9 as the base image for test container

### PR DESCRIPTION
**Pull request by danmanor**

### CODE REVIEW

File: Dockerfile.assisted-installer-build [hunk 1]
```diff
@@ -1,19 +1,34 @@
...
+
+RUN dnf install -y 'dnf-command(config-manager)' && \
+    dnf config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo && \
+    dnf install -y \
+        docker-ce \
+        docker-ce-cli \
+        containerd.io \
+        make \
+        git \
+        openssl-devel \
```
Comment (pastequo) [inline Dockerfile.assisted-installer-build:23]: Did you check if openssl-devel was needed as well in those repo ?

File: Dockerfile.assisted-installer-build [hunk 2]
```diff
@@ -1,19 +1,34 @@
...
+        gcc \
```
Comment (pastequo) [inline Dockerfile.assisted-installer-build:24]: I would also install util-linux explicitly. I think it comes with one of the docker stuff, but without it I had trouble to runner skipper locally

Comment (danmanor) [reply Dockerfile.assisted-installer-build:24]: why would util-linux inside the container would relate to skipper on the host ? I don't need for skipper to work

Comment (pastequo) [reply Dockerfile.assisted-installer-build:24]: https://github.com/Stratoscale/skipper/blob/upstream/skipper/data/skipper-entrypoint.sh#L34

Comment (danmanor) [reply Dockerfile.assisted-installer-build:24]: Oh this is because you are using later version of skipper, I use 2.0.2

Comment (danmanor) [reply Dockerfile.assisted-installer-build:24]: OK lets add it for any case

Comment (danmanor) [reply Dockerfile.assisted-installer-build:24]: I think this problem would occur for you with a lot of other Dockerfiles we use. Maybe add it in different PRs ?

Comment (danmanor) [reply Dockerfile.assisted-installer-build:24]: Although I would recommend using skipper==2.0.2

**Pull request merged**